homeassistant/components/ecobee/climate.py

```diff
@@ -264,6 +264,12 @@ def __init__(self, data, thermostat_index):
         self.thermostat = self.data.ecobee.get_thermostat(self.thermostat_index)
         self._name = self.thermostat["name"]
         self.vacation = None
+        self._last_active_hvac_mode = (
+            ECOBEE_HVAC_TO_HASS[self.thermostat["settings"]["hvacMode"]]
+            if ECOBEE_HVAC_TO_HASS[self.thermostat["settings"]["hvacMode"]]
```

Comment:
Please not use inline if-statements if they span multiple lines. It's not easily readable.

Reply:
I took a look at this again and I think this whole inline if can just be simplified to:
```python
self._last_active_hvac_mode = HVAC_MODE_AUTO
```
Because, when the entity is added to hass we update immediately (`True` gets passed to `async_add_entities`), so if the current value is non-off, self._last_active_hvac_mode will be updated to the correct value.